test/EmailAddressTest.php

```diff
@@ -741,8 +741,8 @@ public function testUseMxCheckBasicValid()
         ];
 
         if (extension_loaded('intl')) {
-            $emailAddresses[] = 'иван@письмо.рф';
```

Comment:
Please place those values which you consider invalid in the appropiate test (testUseMxRecordsBasicInvalid)